definitions/environments/node/flow_v0.261.x-/node.js

```diff
@@ -1307,6 +1313,70 @@ declare module "fs" {
   declare function copyFile(src: string, dest: string, flags?: number, callback: (err: ErrnoError) => void): void;
   declare function copyFileSync(src: string, dest: string, flags?: number): void;
 
+  declare type GlobOptions<WithFileTypes: boolean> = $ReadOnly<{
+    /**
+     * Current working directory.
+     * @default process.cwd()
+     */
+    cwd?: ?string | void,
```

Comment:
```suggestion
    cwd?: string | void,
```
Is there a specific reason for allowing `null` as well?